### ISSUE TITLE
Please Provide the Editable Android Stdudio Project

### ISSUE THREAD

**Issue by venky1995** [sreekanthputta/sreekanthputta.github.io#4]
Hello sir, Thanks for this content on Github. Can u please Provide the Editable Android project of the Android app Sir.... Thank you... If its private to post please send us Email: venkatadri.svr@gmail.com

**Comment by deeptech04** [comment]
he doesn't reply :/

**Comment by venky1995** [comment]
Ok!! bro whats the issue you are facing!! in that> have u done anything with this??

**Comment by deeptech04** [comment]
No, He is the coder, he did everything, I just said he doesn't reply, I don't blame him or anything and I vouch for him for posting the code..  don't take it otherwise..
